pkg/resource/sensitive.go

```diff
@@ -203,22 +202,6 @@ func GetSensitiveParameters(ctx context.Context, client SecretClient, from runti
 			}
 
 			switch v.(type) {
-			case map[string]interface{}:
```

Comment:
What would the behavior be in the case of `map[string]secretRef` added to generated schema but we are dropping support here?

Reply:
We were seeing following error message on the resource:
```
Warning  CannotConnectToProvider  9s (x25 over 23s)  managed/dbforpostgresql.azure.jet.crossplane.io/v1alpha2, kind=server  cannot get a terraform workspace for resource: cannot create a new file producer: cannot get sensitive parameters: cannot unmarshal value from JSON: json: cannot unmarshal string into Go value of type v1.SecretKeySelector
```
Thanks to @sergenyalcin , the issue is fixed now.